docs/rules/miscellaneous/comprehensive-interface.md

```diff
@@ -9,7 +9,7 @@ title:       "comprehensive-interface | Solhint"
 ![Default Severity Badge warn](https://img.shields.io/badge/Default%20Severity-warn-yellow)
 
 ## Description
-Check that all public or external functions are override. This is useful to make sure that the whole API is extracted in an interface.
+Check that all public or external functions are override. This is iseful to make sure that the whole API is extracted in an interface.
```

Comment:
that's from the documentation update, from #389 , which this PR depends on. 

I wanted to get feedback as early as possible, but for that you and other reviewers should look at the changes introduced in the last two commits. 

It seems that it wasn't obvious and I want to be respectful of your time so I'll mark this PR as a draft until #389 is merged, sorry for the mixup

Reply:
sorry... I was just going quickly through the changes, I saw this and just asked....  I'm not assigned full time to this project. And I'm also new to it. I did not approve the former pr because of that. Waiting a few days if fvitorio (the former maintainer) can check it as well... if not, I will do it.
Your contributions are very valuable, sorry if we can't keep up